Source/WebCore/contentextensions/ContentExtensionActions.cpp

```diff
@@ -113,28 +113,35 @@ Expected<ModifyHeadersAction, std::error_code> ModifyHeadersAction::parse(const
     if (!responseHeaders)
         return makeUnexpected(responseHeaders.error());
 
-    return ModifyHeadersAction { WTFMove(*requestHeaders), WTFMove(*responseHeaders) };
+    auto priority = modifyHeaders.getInteger("priority"_s);
+    if (!priority)
```

Comment:
Can we explicitly set a priority of 0?  What about a negative priority?
These also need tests in (ContentExtensionTest, InvalidJSON)

Reply:
I wasn't sure how much validation to do here, since we guarantee that the priority will exist and be positive in Safari code. I will add a tests in (ContentExtensionTest, InvalidJSON).